src/assets/scripts/uio.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright the Fluidic Eleventy copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/fluidic-11ty/raw/master/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/fluidic-11ty/raw/master/LICENSE.md.
+*/
+
+"use strict";
+
+fluid.uiOptions.prefsEditor(".flc-prefsEditor-separatedPanel", {
+    terms: {
+        "templatePrefix": "/lib/infusion/src/framework/preferences/html",
+        "messagePrefix": "/lib/infusion/src/framework/preferences/messages"
+    },
+    "tocTemplate": "/lib/infusion/src/components/tableOfContents/html/TableOfContents.html",
+    "tocMessage": "/lib/infusion/src/framework/preferences/messages/tableOfContents-enactor.json"
+});
```

Comment:
What would the process be for a site instance to configure UIO, that is pass in different options to this function?

Reply:
I'm not sure I understand the question, @jobara. This is a starter project so if you wanted to configure UIO differently you could edit this uio.js file in your project to meet its needs if the default wasn't suitable.

Reply:
Thanks, I forgot this repo is just a template for other repos. 

If not already included, it might be helpful to add some docs in the repos about where to configure UIO and maybe more general about the included scripts in case you didn't want them or want to add more. I suppose it could just mention what is included by default and what needs to be configured for them, and then point at relevant docs elsewhere for how the scripts work. Something like that.

Reply:
@jobara Yeah, that makes sense. Related PR which I don't think you've looked at yet: https://github.com/fluid-project/eleventy-plugin-fluid/blob/041e4802197fdbcdf36086d8f5fa26821a48957a/README.md#shortcodes

Reply:
(For the purposes of this PR, though, I'd like to focus on the JS processing and address documentation around UIO initialization in a separate PR.)